public/app/features/library-panels/state/api.ts

```diff
@@ -48,6 +48,11 @@ export async function getLibraryPanel(uid: string): Promise<LibraryElementDTO> {
   return result;
 }
 
+export async function getLibraryPanelByName(name: string): Promise<LibraryElementDTO> {
+  const { result } = await getBackendSrv().get(`/api/library-elements/byName/${name}`);
```

Comment:
nit: I always forget, but we can type the get request and then get the correct typings
```suggestion
  const { result } = await getBackendSrv().get<LibraryElementDTO>(`/api/library-elements/byName/${name}`);
```